src/libs/actions/Report.js

```diff
@@ -1242,7 +1242,7 @@ Onyx.connect({
  */
 function editReportComment(reportID, originalReportAction, textForNewComment) {
     const parser = new ExpensiMark();
-    const htmlForNewComment = parser.replace(textForNewComment);
+    const htmlForNewComment = parser.replace(textForNewComment, {filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')});
```

Comment:
Let's add a comment explaining why we are removing autolink as it's not obvious for someone not familiar with this issue.

Reply:
@allroundexperts bumping this!

Reply:
@luacmartins Updated.

Reply:
Comment looks good, but linking is failing now

Reply:
@luacmartins Fixed.